src/Command/CreateSnapshotsCommand.php

```diff
@@ -65,6 +79,8 @@ public function execute(InputInterface $input, OutputInterface $output)
                     'mode' => $input->getOption('mode'),
                 ]);
 
+                $this->snapshotManager->save();
```

Comment:
You have to handle (for BC) the case where this property is null.

Reply:
But @VincentLanglet why should it be null? always I'll inject this **snapshotManager**, it's not from outside code, it's in **PageBundle**

Reply:
What if someone use it manually by doing
```
(new CreateSnapshotCommand())->execute(...)
```

Reply:
Make sense 😅